prowler/lib/cli/parser.py

```diff
@@ -220,7 +220,7 @@ def __init_checks_parser__(self):
         group.add_argument(
             "-s", "--services", nargs="+", help="List of services to be executed."
         )
-        group.add_argument(
+        common_checks_parser.add_argument(
```

Comment:
Please include a test in the `tests/lib/cli/parser_test.py` class to verify the use of `--services` with `--severities`.

Reply:
Done!